lib/render/ShaderManager.cpp

```diff
@@ -21,7 +21,7 @@ using std::pair;
 #ifdef NDEBUG
     #if SHADER_AUTORELOAD
         #ifndef WIN32
-            #warning Disabling shader autoreloading
+            //#warning Disabling shader autoreloading
```

Comment:
Why is this commented out?

Reply:
Because it emits a warning.  Do you know of a better alternative?